lib/drive-get-experiment.js

```diff
@@ -0,0 +1,52 @@
+const Hyperdrive = require('hyperdrive')
+const Corestore = require('corestore')
+const Experiment = require('./experiment')
+
+const main = async () => { 
```

Comment:
The experiment should not contain a main method

Reply:
removed.